exercises/es/exc_03_11.py

```diff
@@ -1,22 +1,22 @@
 import spacy
 from spacy.tokens import Span
 
-nlp = spacy.load("en_core_web_sm")
+nlp = spacy.load("es_core_news_sm")
 
 
 def get_wikipedia_url(span):
     # Obtén la URL de Wikipedia si el span tiene uno de los siguientes labels
-    if ____ in ("PERSON", "ORG", "GPE", "LOCATION"):
+    if ____ in ("PER", "ORG", "LOC"):
         entity_text = span.text.replace(" ", "_")
-        return "https://en.wikipedia.org/w/index.php?search=" + entity_text
+        return "https://es.wikipedia.org/w/index.php?search=" + entity_text
 
 
 # Añade la extensión del Span, wikipedia_url, usando el getter get_wikipedia_url
 ____.____(____, ____=____)
 
 doc = nlp(
-    "In over fifty years from his very first recordings right through to his "
-    "last album, David Bowie was at the vanguard of contemporary culture."
+    "Antes de finalizar 1976, el interés de David Bowie en la floreciente escena musical alemana,"
```

Comment:
My initial reaction was, hmm, not sure if the example here is good because unlike all the other examples, it produces two Wikipedia links for Alemania and David Bowie. But maybe that's not so bad actually?

Reply:
I left it as it is. I thought it was nice to show that it could handle two different categories of entities.